src/test/java/com/google/step/youtube/DescriptionParserTest.java

```diff
@@ -180,14 +182,16 @@ public void parse_snippetTruncateEnd() {
                 + "at the top of the homepage, and type in ROOSTER to claim";
         String desc = truncatedSnippet + " your special offer).";
         assertThat(DescriptionParser.parse(desc), equalTo(Arrays.asList(
-                    OfferSnippet.create("http://stamps.com", truncatedSnippet + " ..."))));
+                OfferSnippet.create("http://stamps.com", String.format("%s %s", 
```

Comment:
Take a look at go/tott/510

You probably just want string literals for the expected values.  don't use String.format()

Reply:
Removed the formatted strings in tests.